src/test/java/uk/gov/digital/ho/hocs/migration/JSONValidate.java

```diff
@@ -49,6 +48,36 @@ public void testInvalidCaseData() throws Exception {
         }
     }
 
```

Comment:
If we're adding otherCorrespondents to the schema, perhaps we should add some tests to that too. But our focus at the moment is only a single correspondent so maybe we should leave it our of the schema for now?

Reply:
As agreed, we decided to only focus on primary correspondents for this ticket, other correspondents have now been removed.